bsp_diff/common/vendor/intel/fw/evmm/0001-Use-prebuilts-clang-instead-of-gcc-to-build-evmmpack.patch

```diff
@@ -0,0 +1,55 @@
+From 09a06726a6d710c4411ffe5b7e9c448a0379379c Mon Sep 17 00:00:00 2001
```

Comment:
why not submit to upstream? please PR to: https://github.com/intel/ikgt-core/tree/trusty